### PR TITLE
Hello

### PR DESCRIPTION
**Pull request by chrismtocha**
Hello Adele

### CODE REVIEW

File: lyrics/pop/Hello.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Hello, it's me
```
Comment (raagaakkineni) [inline lyrics/pop/Hello.txt:1]: is it really?

File: lyrics/pop/Hello.txt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+I was wondering if after all these years you'd like to meet
+To go over everything
+They say that time's supposed to heal ya, but I ain't done much healing
+Hello, can you hear me?
```
Comment (raagaakkineni) [inline lyrics/pop/Hello.txt:5]: No, you are on mute

**Review by raagaakkineni — COMMENTED** [review]
comments added

File: lyrics/pop/Hello.txt [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+I must've called a thousand times (thousand times)
+To tell you I'm sorry for everything that I've done
+But when I call, you never seem to be home
+Hello from the outside (outside)
+At least I can say that I've tried (I've tried)
+To tell you I'm sorry for breaking your heart
+But it don't matter, it clearly doesn't tear you apart anymore
+Ooh (lows, lows, lows, lows), anymore
+(Highs, highs, highs, highs)
+Ooh (lows, lows, lows, lows), anymore
```
Comment (mcortinat) [inline lyrics/pop/Hello.txt]:
```suggestion
OHH, ANYMORE
```

File: lyrics/pop/Hello.txt (hunk 2 above)
Comment (chrismtocha) [reply lyrics/pop/Hello.txt:5]: Unmuted

**Pull request merged**